ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/testFiles/TestPtpConfig/sourcePolicies/TestPtpConfig.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: ptp.openshift.io/v1
```

Comment:
Is there a reason to duplicate the source files ? can we just refer to what is already under ztp/source-policy-crs ?

Reply:
trying to keep the CRs we use for testing separate.. as we would eventually add invalid CRs as well to go over failure cases.. try and maintain distinction between configuration and test config..